modules/concept-docs/pages/certificate-based-authentication.adoc

```diff
@@ -1 +1 @@
-include::6.5@sdk:pages:partial$certificates.adoc[]
+include::7.0@sdk:pages:partial$certificates.adoc[]
```

Comment:
Seems like a good candidate for `{version-server}` this.

Same for [data-services.adoc](https://github.com/couchbase/docs-sdk-c/pull/109/files#diff-febe4b7248858860d4cc744ee24fb0c5e506ffc53a70aa0f4e231a7c090d320eR1)

Reply:
absolutely - but as it was driveby, and we haven't done this work in general for lcb docs, I thought it was better to keep it simple for this commit.